src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -2094,11 +2094,24 @@ int32_t guest_path_finding(Guest* peep)
         int32_t select = peep->no_of_rides % numEntranceStations;
         while (select > 0)
         {
-            closestStationNum = bitscanforward(entranceStations);
-            entranceStations &= ~(1 << closestStationNum);
-            select--;
+            for (size_t i = 0; i < entranceStations.size(); i++)
```

Comment:
This was done because MAX_STATION will be raised to 255, which makes it too large to stuff into a uint8_t or even a uint64_t.